### DISCUSSION TITLE
The Multi Lingual MSI works for all languages but Spanish

### DISCUSSION THREAD

**Discussion by CuriusSai** [wixtoolset/issues#6833] [Questions]
## Triage

# Bugs

If this issue is a bug:

* Which version of WiX are you building with?

> (Wix 3.11.01701)

* Which version of Visual Studio are you building with (if any)?

> (Microsoft Visual Studio Professional 2022 (64-bit) - Current
Version 17.1.4)

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> (WiX Toolset Visual Studio Extension 1.0.0.18)

* Which version of .NET are you building with?

> (.NET version: 4.6.1, 4.7.2, 4.8)

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> (Windows version: 21H2)

* Describe the problem and the steps to reproduce it.

> (1. Add .xwl for each language (en-us, es-es, fr-fr, de-de, ja-jp, add numeric languages codes in .wxs file, use (!loc.Language) in .wxs file)
2. Build the project. To generate separate MSI file for each language
3. Change system local to German - Run the Installer, see German texts, run the Spanish installer it shows everything in English!!!)

* Describe the behavior you expected and how it differed from the actual behavior.

> (The Spanish installer show test in Spanish and not in English.)

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh]. You might also consider asking your question on the [wix-users mailing list][ml] or posting on [Stack Overflow][so].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
[ml]: http://wixtoolset.org/documentation/mailinglist/
[so]: https://stackoverflow.com/questions/tagged/wix
